keras_trainer/data_generators.py

```diff
@@ -712,12 +742,14 @@ class EnhancedImageDataGenerator(ImageDataGenerator):
     """
 
     This is a modification of the Keras class ImageDataGenerator that includes some extra functionalities.
-    The image data generator has a new option `random_crop_size` that if specified will perform random crops prior to
+    The image data generator has a new options `random_crop_size` and `custom_crop` that if specified will perform random crops prior to
     resize the image to the specified target size.
 
     - random_crop_size:  Size of the random crop. Either a percentage of the original image (0,1) that will do square
         crop, a fixed size (tuple), or integer where the value will set equally to both dimensions.
         target_size: tuple of integers, dimensions to resize input images to.
+    - custom_crop: A boolean, if True custom crops will be done according to the value in 'z_col'. This has been
```

Comment:
I believe Albert uses ` :
```suggestion
    - custom_crop: A boolean, if True custom crops will be done according to the value in `z_col`. This has been
```